introduction.md

```diff
@@ -1 +1,6 @@
-Hello World. My name is Konstantin. And I love JavaScript path of CodeSchool!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?